tests/benchmarks/latency_measure/CMakeLists.txt

```diff
@@ -2,15 +2,25 @@ cmake_minimum_required(VERSION 3.13.1)
 set(small_freq_divider_frdm_k64f TRUE)
 set(small_freq_divider_arduino_due TRUE)
 set(small_freq_divider_qemu_cortex_m3 TRUE)
+set(small_freq_divider_stm32f4_disco TRUE)
 
-if(small_freq_divider_${BOARD})
-  set(CONF_FILE prj_small_freq_divider.conf)
-else()
-  set(CONF_FILE prj.conf)
-endif()
+set(CONF_FILE prj.conf)
 
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(latency_measure)
 
+# eliminate timer interrupts during the benchmark
```

Comment:
Why not just check if(${CONFIG_CORTEX_M_SYSTICK}) and set or do the logic associated with small_freq_divider so we don't have to add that flag.